mainhandler/imageregistryhandler.go

```diff
@@ -800,7 +800,13 @@ func (registryScan *registryScan) setRegistryInfoFromConfigMap(registryInfo *arm
 	registryInfo.Exclude = registryConfig.Exclude
 }
 
-func getRegistryScanSecrets(k8sAPI *k8sinterface.KubernetesApi, namespace, secretName string) ([]k8sinterface.IWorkload, error) {
+// KubernetesApiSecrets is an interface for getting workloads from k8s api
+type IWorkloadsGetter interface {
```

Comment:
Is this in use?

Reply:
Yes, to make the code testable. see `vulnscan_test.go`